src/components/Greeting.js

```diff
@@ -38,4 +38,4 @@ function Greeting() {
   );
 }
 
-export default Greeting;
\ No newline at end of file
+export default Greeting;
```

Comment:
Good work implementing your front-end repository! 

You are really near to connecting it, I think that you are having problems with routing. Kindly check if you are receiving correctly your data in the front end! 


![image](https://github.com/Luchy-B/hello-react-front-end/assets/93630700/5207cd89-8779-452f-b2ab-12ec025af8cc)